test/system/030-run.bats

```diff
@@ -1197,6 +1198,12 @@ search               | $IMAGE           |
         run_podman 125 $command --authfile=$bogus $args
         assert "$output" = "Error: checking authfile: stat $bogus: no such file or directory" \
            "$command --authfile=nonexistent-path"
+
+        if [[ "$command" != "logout" ]]; then
+           REGISTRY_AUTH_FILE=$bogus run_podman ? $command $args
+           assert "$output" != "Error: checking authfile: stat $bogus: no such file or directory" \
+              "$command --authfile=nonexistent-path"
+        fi
```

Comment:
Suggestion:
```bash
           assert "$output" !~ "checking authfile: " \
                  "$command with REGISTRY_AUTH_FILE=nonexistent-path"

```
Reason: (1) I don't like exact `!=` checks on error messages. What if you get "Error: checking authfile: .... plus more output"? and (2) test name/title is misleading